zboxapi/sdk.go

```diff
@@ -13,8 +13,34 @@ import (
 	"github.com/0chain/gosdk/core/encryption"
 	"github.com/0chain/gosdk/core/resty"
 	"github.com/0chain/gosdk/core/sys"
+	"github.com/0chain/gosdk/core/zcncrypto"
 )
 
+func init() {
+	if sys.Sign == nil {
```

Comment:
we have init function in `zboxcore/client/entity.go` which initializes the same value like this, it is possible that it can be reinitialized since the order of init functions is not defined, please use once() to allow only single initialization

```
func init() {
	client = &Client{
		Wallet: &zcncrypto.Wallet{},
	}

	sys.Sign = signHash
	// initialize SignFunc as default implementation
	Sign = func(hash string) (string, error) {
		return sys.Sign(hash, client.SignatureScheme, GetClientSysKeys())
	}

	sys.Verify = VerifySignature
	sys.VerifyWith = VerifySignatureWith
}
```

Reply:
in zboxapi, we don't import zbox package. let me try to improve it.

Reply:
used local `signHash` instead of sys.SignFunc